app/views/users/show.html.erb

```diff
@@ -9,41 +9,43 @@
   </div>
 </div>
 <br>
-<div class="col-md-8">
-    <div class="row">
-        <div class="col-md-6">
-            <label><%= t('users.name').capitalize() %>:</label>
-        </div>
-        <div class="col-md-6">
-            <p><%= @user.name %></p>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-6">
-            <label><%= t('users.email').capitalize() %>:</label>
-        </div>
-        <div class="col-md-6">
-            <p><%= @user.email %></p>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-6">
-            <label><%= t('users.role').capitalize() %>:</label>
-        </div>
-        <div class="col-md-6">
-            <p><%= @user.role %></p>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-md-6">
-            <label><%= t('users.company').capitalize() %>:</label>
-        </div>
-        <div class="col-md-6">
-            <p><%= @user.company.name %></p>
-        </div>
-    </div>
-    <br>
-    <div>
-      <%= button_to t('shared.update_resource', resource_name: t('shared.user')), edit_user_registration_path, method: :get, class: 'btn btn-info' %>
-    </div>
-</div>
+<div class="container">
+  <div class="col-md-8">
+      <div class="row">
+          <div class="col-md-6">
+              <label><%= t('users.name').capitalize() %>:</label>
+          </div>
+          <div class="col-md-6">
+              <p><%= @user.name %></p>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-md-6">
+              <label><%= t('users.email').capitalize() %>:</label>
+          </div>
+          <div class="col-md-6">
+              <p><%= @user.email %></p>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-md-6">
+              <label><%= t('users.role').capitalize() %>:</label>
+          </div>
+          <div class="col-md-6">
+              <p><%= @user.role %></p>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-md-6">
+              <label><%= t('users.company').capitalize() %>:</label>
+          </div>
+          <div class="col-md-6">
+              <p><%= @user.company.name %></p>
+          </div>
+      </div>
```

Comment:
Do we need to display company?

Reply:
Yes you are right. He already knows that he has login into it.